### ISSUE TITLE
Killzone Liberation DEMO file is not loading (crashes upon start)

### ISSUE THREAD

**Issue by VosslerXIII** [hrydgard/ppsspp#14790]
EDIT: Solution: Rename folder in PSP/GAME to UCUS98646-Demo and it'll boot.

### Game or games this happens in

UCUS-98646 - Killzone Liberation
Link to the DEMO - https://archive.org/details/UCUS98646_Demo

### What area of the game / PPSSPP

At the very start when you try to load the game 

### What should happen

the game should start working

### Logs

![Capture](https://user-images.githubusercontent.com/51298820/131926289-a93b19f1-b7a8-4c6a-bb86-c4c7f51a8f55.JPG)

### Platform

Windows

### Mobile phone model or graphics card

GTX 1070ti

### PPSSPP version affected

1.11.3

### Graphics backend (3D API)

Direct3D 11

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by ANR2ME** [comment]
Wow that's a lot of memory to be allocated O.o

**Comment by hrydgard** [comment]
Kinda looking like one of those games that tries to probe for the biggest possible allocation they can make and then does it, but somehow this never succeeds?

**Comment by XiaomiRedmi9** [comment]
File is corrupted?
![Screenshot_2021-09-03-16-49-23-387_org ppsspp ppssppgold](https://user-images.githubusercontent.com/78680959/131978125-501f64bd-7dd2-4c9d-8ab9-1e4ad041032c.jpg)

**Comment by ANR2ME** [comment]
> Failed to allocate

The first "failed to allocate" text is on line 804, may be you're searching it with case-sensitive :)

**Comment by XiaomiRedmi9** [comment]
I try a different demo file for Killzone Liberation and it works :P
![Screenshot_2021-09-03-17-10-08-681_org ppsspp ppsspp](https://user-images.githubusercontent.com/78680959/131981026-ecb4ec55-2d35-4476-9abe-53d4ee1b13ad.jpg)
![Screenshot_2021-09-03-17-09-54-192_org ppsspp ppsspp](https://user-images.githubusercontent.com/78680959/131981044-515e4f5d-c334-458a-949b-07f02b7dd48f.jpg)

Source ~ https://playdreamcreate.com/

**Comment by ANR2ME** [comment]
May be only US version that have this issue?

**Comment by VosslerXIII** [comment]
I posted this issue because this game is one of the most unique ones due to the DLC behavior (Chapter 5), Web browser requirements, and the whole code itself. 
The EU version used to be on on a separate download resources. However, the US version has been released only on UMD discs (https://www.ebay.com/itm/193802405362 example); the archive.org dumped one of those UMD discs and there are some encryption involved in it.

**Comment by unknownbrackets** [comment]
It logs this:

```
=== Memory Status ===
32MB Mode
Kernel:       4194304
Volatile:     4194304
Available:   25165824
---------------------
Kernel
Total Free:    786432
Max Free:      786432
---------------------
PIGS memory manager
OS Overhead:   802816
Application:  6714368
Heap:               0
Stack:          65536 (0 safety guard)
---------------------
Total Used:   7582720
Total Free:  18369536
App Heap:    17583104
=====================
```

It does end up with allocated memory, so it doesn't seem like that's directly the cause of it exiting.  Tempting to infer that since there's nothing else logged.

If I skip the sceKernelExitGame, it ends up with invalid memory access.  So digging further, it crashed after this (DEBUG level):

```
sceIoGetstat(ms0:/PSP/GAME/UCUS98646-Demo/KZL.DRM, 08e7ac98) : FILE NOT FOUND
```

After renaming it from "UCUS98646" to "UCUS98646-Demo", it works fine.  I suspect this isn't actually a bug, and the game would probably ragequit on a real PSP with the wrong directory name.  Haven't tried on a real PSP, though.

-[Unknown]
